shared-bindings/i2cperipheral/__init__.c

```diff
@@ -84,22 +84,22 @@
 //|   0xaa
 //|
 //| .. warning::
-//|    I2CSlave makes use of clock stretching in order to slow down the master.
-//|    Make sure the I2C master supports this.
+//|    I2CPeripheral makes use of clock stretching in order to slow down the main device.
+//|    Make sure the main I2C device supports this.
```

Comment:
I feel like the use of device is a little confusing here and "main" doesn't really help. What do you think about saying "host" instead? My preference would be to use "main" only in MOSI/MISO context. Everything else can be host/peripheral and "selected peripheral".